approach.md

```diff
@@ -28,7 +36,7 @@ the practice of Research Computing Facilitation.
 </div>
 <div class="col-auto">
 <div class="rounded py-1">
-<a class="button rounded btn-primary p-2" href="/htc.html">Explore What is HTC?</a>
+<a class="button rounded btn-primary p-2" href="/htc.html">What is HTC?</a>
```

Comment:
It is more common to use buttons for actions. I would link this when you reference high throughput computing above.